test_outputs/witnesses/function_missing.output.ron

```diff
@@ -0,0 +1,47 @@
+{
+    "./test_crates/feature_flags_validation/": [
+        (
+            filename: "src/lib.rs",
+            begin_line: 4,
+            hint: "use feature_flags_validation::foo_becomes_gated;\nfoo_becomes_gated(...);",
```

Comment:
it's mentioned in the comment in the test code, but prettier (multiline) strings are blocked on the release of ron v0.9 (which although there is an alpha, doesn't seem to be happening anytime soon due to maintainer overburden).  it may be worth (albeit hacky), doing a `str.replace("\\n", "\n")` after serializing to a string for better diffs.